root/admin/wikidoc/WikiDocIndex.js

```diff
@@ -0,0 +1,194 @@
+/*
+ * @flow
+ * Copyright (C) 2020 MetaBrainz Foundation
+ *      /*  ((n   *'.
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+import {defaults} from 'lodash';
+
+import Layout from '../../layout';
+import Table from '../../components/Table';
+import {withCatalystContext} from '../../context';
+
+import type {WikiDocT} from './types';
+
+type PropsT = {
+  +$c: CatalystContextT,
+  +pages: $ReadOnlyArray<WikiDocT>,
+  +updates_required: boolean,
+  +wiki_server: string,
+  +wiki_unreachable: boolean,
+};
+
+const WikiDocTable = withCatalystContext(({
+  $c,
+  pages,
+  updates_required,
+  wiki_server,
+}: PropsT) => {
+  const columns = React.useMemo(
+    () => {
+      const nameColumn = {
+        Cell: ({cell: {value}}) => (
+          <a href={'/doc/' + encodeURIComponent(value)}>{value}</a>
+        ),
+        Header: N_l('Page name'),
+        accessor: 'id',
+        cellProps: {className: 'title'},
+        headerProps: {className: 'c'},
```

Comment:
Sorry, that was not intended, left align looks better because of subpages, see:

|foo | bar |
|:--- | --- |
|baz | bim |
|baz/zoo | bim |

versus

| foo | bar |
| :-: | --- |
| baz | bim |
| baz/zoo | bim |

Reverted in https://github.com/metabrainz/musicbrainz-server/pull/1461/commits/126dd6225650b6eb0f4096854b7ed63b4265b12e.